app/models/user_extension/authenticated_user.rb

```diff
@@ -92,7 +66,7 @@ def remember_token?
   # These create and unset the fields required for remembering users between browser closes
   def remember_me
     self.remember_token_expires_at = 2.weeks.from_now.utc
-    self.remember_token            = encrypt("#{email}--#{remember_token_expires_at}")
+    self.remember_token            = SecureRandom.hex
```

Comment:
@elijh, I moved the 'encrypt' function to UserExtension::LegacyPasswords.legacy_hash. So instead of a hash i use a purely random 128 bit entropy string here.
Any downsides you can think of?